src/components/presentational/Action/Action.stories.tsx

```diff
@@ -1,51 +1,64 @@
 ﻿import React from "react";
-import { ComponentStory, ComponentMeta } from "@storybook/react";
 import Action, { ActionProps } from "./Action";
-import Layout from "../Layout"
+import Layout from "../Layout";
 import './Action.stories.css';
 import { FontAwesomeSvgIcon } from "react-fontawesome-svg-icon";
 import { faFile } from "@fortawesome/free-solid-svg-icons";
+import { Meta, StoryObj } from "@storybook/react";
 
-export default {
+const meta: Meta<typeof Action> = {
 	title: "Presentational/Action",
 	component: Action,
 	parameters: {
-		layout: 'fullscreen',
+		layout: 'fullscreen'
 	}
-} as ComponentMeta<typeof Action>;
+};
 
-const Template: ComponentStory<typeof Action> = (args: ActionProps) =>
+export default meta;
+type Story = StoryObj<typeof Action>;
+
+const render = (args: ActionProps) =>
 	<Layout colorScheme="auto">
 		<Action {...args} />
 	</Layout>;
 
-export const StartSurvey = Template.bind({});
-StartSurvey.args = {
-	title: "Baseline Survey",
-	subtitle: "Tap here to start your baseline survey",
-	onClick: () => alert("Clicked")
+
+export const StartSurvey: Story = {
+	args: {
+		title: "Baseline Survey",
+		subtitle: "Tap here to start your baseline survey",
+		onClick: () => alert("Clicked")
+	},
+	render: render
 }
 
-export const WithClass = Template.bind({});
-WithClass.args = {
-	title: "Baseline Survey",
-	subtitle: "Tap here to start your baseline survey",
-	className: "action-story-primary",
-	onClick: () => alert("Clicked")
+export const WithClass: Story = {
+	args: {
+		title: "Baseline Survey",
+		subtitle: "Tap here to start your baseline survey",
+		className: "action-story-primary",
+		renderAs: "div",
```

Comment:
It looks like the `action-story-primary` class pre-dated this PR, but it wasn't showing up before. The contrast/readability is awful. I think it would be a better example if this used one of our standard colors.

There are a few others later too using random blues and yellows. I know that they're just examples, but they're also meant to illustrate the cool things that you can do with the components. It's just a little jarring when we've taken pains in so many other examples to make them look aesthetic and realistic.

(Doesn't have to be done in this PR - we can spin off a separate issue if you think it's worth doing.)

Reply:
I'm updating it to use --mdhui-color-primary